src/api/Feed.js

```diff
@@ -749,13 +750,16 @@ class Feed extends Base {
                 limit: 1,
                 next_marker: null,
             },
-            task_type: 'GENERAL',
+            task_type: taskType,
             status: TASK_NEW_NOT_STARTED,
         };
 
         this.addPendingItem(this.id, currentUser, pendingTask);
 
-        const taskPayload = { name: message, due_at: dueAtString };
+        const taskPayload: TaskPayload = { name: message, due_at: dueAtString };
+        if (taskType) {
```

Comment:
According to the flow typing, it seems like `taskType` is required, is this if check needed?

Reply:
will update doc to make it optional

Reply:
let's make it required - you should have to know what type of task you create

Reply:
ah yeah that makes sense, I'll go ahead and make it required